server_addon/maya/client/ayon_maya/plugins/publish/collect_yeti_rig.py

```diff
@@ -274,19 +275,18 @@ def get_sequence(self, filepath, pattern="%04d"):
             list: file sequence.
 
         """
-        import clique
-
-        escaped = re.escape(filepath)
-        re_pattern = escaped.replace(pattern, "-?[0-9]+")
-
+        filename = os.path.basename(filepath)
+        re_pattern = re.escape(filename)
+        re_pattern = re_pattern.replace(re.escape(pattern), "-?[0-9]+")
         source_dir = os.path.dirname(filepath)
-        files = [f for f in os.listdir(source_dir)
-                 if re.match(re_pattern, f)]
-
-        pattern = [clique.PATTERNS["frames"]]
-        collection, remainder = clique.assemble(files, patterns=pattern)
-
-        return collection
+        files = [f for f in os.listdir(source_dir) if re.match(re_pattern, f)]
+        collection, _remainder = clique.assemble(
+            files,
+            patterns=[clique.PATTERNS["frames"]],
+            minimum_items=1)
+        files = [texture for texture_collection in collection
+                 for texture in texture_collection]
+        return files
```

Comment:
You are now potentially merging multiple collections - which I suppose is not intended?

```suggestion
        collections, _remainder = clique.assemble(
            files,
            patterns=[clique.PATTERNS["frames"]],
            minimum_items=1)
            
        if len(collections) > 1:
            raise ValueError(f"Multiple collections found for {files}. This is a bug.")

        return list(collections[0])
```